web/src/main/java/org/wso2/testgrid/web/api/TestPlanService.java

```diff
@@ -174,17 +174,17 @@ public Response getLogContent(@PathParam("id") String id, @QueryParam("truncate"
                                                                 artifactDownloadable.readArtifact(bucketKey);
             return Response.status(Response.Status.OK).entity(truncatedInputStreamData).build();
         } catch (TestGridDAOException e) {
-            String msg = "Error occurred while fetching the TestPlan by id : '" + id + "'";
+            String msg = "Error occurred while fetching the TestPlan by id : '" + id + "' " + e.getMessage();
```

Comment:
We are passing the message along with the exception when logging. Shall we append `e.getMessage()` when sending the response?

Reply:
I don't think it would make much difference. Some people prefer to have the cause message in the first line itself.